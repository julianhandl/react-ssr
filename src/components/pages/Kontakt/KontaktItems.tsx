import React from 'react';
import './KontaktItemsStyles.scss';

const KontaktItems : React.FunctionComponent = () => {
    return (
        <div className="container">
            <div className="kontakt-items">
                <div className="kontakt-items__container">
                    <div className="kontakt-item">
                        <h2>E-Mail</h2>
                        <a href="mailto:office@partypartner.at">office@partypartner.at</a>
                    </div>
                    <div className="kontakt-item">
                        <h2>Telefon</h2>
                        <a href="tel:+4367763167169">+43 677 631 671 69</a>
                    </div>
                    <div className="kontakt-item">
                        <h2>Whatsapp</h2>
                        <a target="_blank" href="https://wa.me/4367763167169">+43 677 631 671 69</a>
                    </div>
                </div>
                <div className="kontakt-item">
                    <h2>Adresse</h2>
                    <p>
                        Julian Handl<br />
                        Dreihäuslweg 2<br />
                        3253 Erlauf<br />
                        Österreich
                    </p>
                </div>
            </div>
        </div>
    );
};

export default KontaktItems;