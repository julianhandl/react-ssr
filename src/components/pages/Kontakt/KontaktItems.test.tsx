import React from "react";
import { mount } from "enzyme";
import KontaktItems from "./KontaktItems";

describe("KontaktIntro", () => {
    it("should render normal", () => {
        const wrapper = mount(<KontaktItems />);
        expect(wrapper).toMatchSnapshot();
    })
})