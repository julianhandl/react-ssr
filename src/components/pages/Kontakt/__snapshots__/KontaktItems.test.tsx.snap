// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`KontaktIntro should render normal 1`] = `
<KontaktItems>
  <div
    className="container"
  >
    <div
      className="kontakt-items"
    >
      <div
        className="kontakt-items__container"
      >
        <div
          className="kontakt-item"
        >
          <h2>
            E-Mail
          </h2>
          <a
            href="mailto:office@partypartner.at"
          >
            office@partypartner.at
          </a>
        </div>
        <div
          className="kontakt-item"
        >
          <h2>
            Telefon
          </h2>
          <a
            href="tel:+4367763167169"
          >
            +43 677 631 671 69
          </a>
        </div>
        <div
          className="kontakt-item"
        >
          <h2>
            Whatsapp
          </h2>
          <a
            href="https://wa.me/4367763167169"
            target="_blank"
          >
            +43 677 631 671 69
          </a>
        </div>
      </div>
      <div
        className="kontakt-item"
      >
        <h2>
          Adresse
        </h2>
        <p>
          Julian Handl
          <br />
          Dreihäuslweg 2
          <br />
          3253 Erlauf
          <br />
          Österreich
        </p>
      </div>
    </div>
  </div>
</KontaktItems>
`;
